### PR TITLE
CEDS-2170 remove gateway timeout response

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/views/components/ilequery/response_buttons.scala.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
   <div>
       @govukButton(Button(content = Text(messages("ileQueryResponse.links.manageConsignment")), href = Some(routes.ChoiceController.displayPage().url)))
   </div>
-  <div class="govuk-body govuk-!-margin-bottom-9">
-      @link(messages("ileQueryResponse.links.findConsignment"), controllers.ileQuery.routes.IleQueryController.displayQueryForm)
+  <div>
```
Comment (SteveSugden) [inline app/views/components/ilequery/response_buttons.scala.html]: This section needs the space at the bottom to look right.  Was changed when the buttons moved from the bottom of the screen to the middle

File: test/unit/views/IleQueryDucrResponseViewSpec.scala [hunk 2]
```diff
@@ -50,8 +55,52 @@ class IleQueryDucrResponseViewSpec extends ViewSpec with Injector {
...
     }
 
-    "render parent" in {
-      view(parent = Some(parentInfo)).getElementById("parentConsignment") must containMessage("ileQueryResponse.parent")
+    "render default status of entry" in {
+      summaryElement(view(), 1).text must be("")
+    }
+
+    "render empty status of entry" in {
+      summaryElement(view(ducrInfo.copy(entryStatus = Some(status.copy(soe = None)))), 1).text must be("")
```
Comment (SteveSugden) [inline test/unit/views/IleQueryDucrResponseViewSpec.scala]: All these tests have been moved out of the ducr/mucr view specs and into tests for the individual components.  Also the some of the tests are no longer valid - each missing code now displays no row, not empty value

File: test/unit/views/IleQueryMucrResponseViewSpec.scala [hunk 3]
```diff
@@ -74,5 +80,46 @@ class IleQueryMucrResponseViewSpec extends ViewSpec with Injector {
...
+    )
+
+    "render parent consignment link" in {
+      parentConsignmentElement(viewWithParent, 0).getElementsByClass("govuk-link").first() must haveHref(
+        controllers.ileQuery.routes.IleQueryController.getConsignmentInformation("parentUcr")
+      )
+      parentConsignmentElement(viewWithParent, 0) must containText("parentUcr")
+    }
+
+    "render parent consignment route" in {
```
Comment (SteveSugden) [inline test/unit/views/IleQueryMucrResponseViewSpec.scala]: Ditto

File: test/unit/views/IleQueryDucrResponseViewSpec.scala (hunk 2 above)
Comment (MaciejRewera) [reply test/unit/views/IleQueryDucrResponseViewSpec.scala]: Copied the tests as they are on GitHub. Should be fine now.
